apps/next/src/pages/Settings/components/Home/components/SettingsCard.tsx

```diff
@@ -13,11 +14,11 @@ export const SettingsCard: FC<SettingsCardProps> = ({
   description,
   ...cardProps
 }) => {
+  const theme = useTheme();
   return (
     <StyledCard {...cardProps}>
       <Stack gap={0.5}>
-        {/* TODO: find a common ground with UX team so we don't have to override the fontWeight*/}
-        <Typography variant="h5" fontWeight="700 !important">
+        <Typography variant="h5" fontWeight={theme.typography.fontWeightBold}>
```

Comment:
You should be able to use `fontWeight="fontWeightBold"`, no need to `useTheme()`.